dockerfiles/verify_packages/installer/test_install_tracer_appsec_latest_piecewise.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env sh
+
+set -e
+
+. "$(dirname ${0})/utils.sh"
+
+# Initially no ddtrace/ddappsec
+assert_no_ddtrace
+assert_no_ddappsec
+
+# Install first the latest tracer
+php dd-library-php-setup.php --php-bin php --no--appsec
```

Comment:
here and on line 19 there it should be `--no-*`, why is the test not failing?

Reply:
`assert_no_ddappsec` is buggy. I've fixed this.